metatron-app/metatron-discovery
# #2008 improve UX for function selection area on user-defined field view

### Description
Improve UX for function selection area on user-defined field view

**Related Issue** : #2008 



### How Has This Been Tested?
1. Go to workbook and create or edit Chart.
2. Move to create custom-filed view.
3. Check tUX for function selection area on user-defined field view.

#### Need additional checks?
Check both language. (If your language is Chinese, it is displayed in English.)

### Types of changes



### Checklist:


### Additional Context
